descriptors/ModuleDescriptor-template.json

```diff
@@ -1,29 +1,76 @@
 {
   "id": "${artifactId}-${version}",
-  "name": "The blueprint module that could be used as a template for folio spring based backend modules.",
+  "name": "Search Module",
   "provides": [
     {
-      "id": "spring-template",
-      "version": "1.0",
-      "handlers": []
+      "id": "indexes",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/search/index/indices",
+          "permissionsRequired": ["search.index.indexes.item.post"]
```

Comment:
Should we make it consistent, `indices` vs `indexes`?

Reply:
Yes, agree, renamed to `indices`.